Dependencies/xr/Source/OpenXR/include/IXrContext.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <openxr/openxr.h>
+#include <openxr/openxr_platform.h>
+#include <XrUtility/XrExtensions.h>
+
+#if defined(__CYGWIN32__)
+    #define OPENXR_CONTEXT_INTERFACE_API __stdcall
+#elif defined(WIN32) || defined(_WIN32) || defined(__WIN32__) || defined(_WIN64) || defined(WINAPI_FAMILY)
+    #define OPENXR_CONTEXT_INTERFACE_API __stdcall
+#elif defined(__MACH__) || defined(__ANDROID__) || defined(__linux__) || defined(LUMIN)
+    #define OPENXR_CONTEXT_INTERFACE_API
+#else
+    #define OPENXR_CONTEXT_INTERFACE_API
+#endif
+
+typedef struct IXrContext
+{
+    virtual bool OPENXR_CONTEXT_INTERFACE_API IsInitialized() const = 0;

```

Comment:
will the consumer side use this bool differently than simply test "Instance() == 0"?

Reply:
we currently use this to ensure the session is running